client/apps/edit/components/section_video/icons.jade

```diff
@@ -0,0 +1,2 @@
+.remove
+  include ../../public/icons/edit_section_remove.svg
```

Comment:
not for this PR and more just a random thought, but if we're comfortable with continuing this pattern of including svg's, it would be nice to create a single `icons.jade` file that maps all icons so we don't have to keep creating these files.